command/expand_test.go

```diff
@@ -328,6 +329,86 @@ func TestExpandSource_Do_Not_Follow_Symlinks(t *testing.T) {
 	assert.Equal(t, []string{workdirJoin}, expected)
 }
 
+func TestExpandSourceWithRawFlagTrue(t *testing.T) {
+	// windows does not have support filenames with *.
+	if runtime.GOOS == "windows" {
+		t.Skip()
+	}
+	t.Parallel()
+	ctx := context.Background()
+	fileContent := "CAFEBABE"
+	folderLayout := []fs.PathOp{
+		fs.WithDir(
+			"a",
+			fs.WithFile("file*.txt", fileContent),
+			fs.WithFile("file*1.txt", fileContent),
+			fs.WithFile("file*2.txt", fileContent),
+			fs.WithFile("file*file*.txt", fileContent),
+		),
+		fs.WithDir(
+			"c",
+			fs.WithFile("file*5*5.txt", fileContent),
+		),
+	}
+
+	workdir := fs.NewDir(t, t.Name(), folderLayout...)
+	defer workdir.Remove()
+
+	workdirUrl, _ := url.New(workdir.Join("a/file*.txt"))
+
+	//do not follow symbolic links
+	ch, _ := expandSource(ctx, storage.NewLocalClient(storage.Options{}), false, workdirUrl, true) // set raw flag to true
+	assert.Equal(t, len(ch), 1)
+	var expected []string
+	for obj := range ch {
+		expected = append(expected, obj.URL.Absolute())
+	}
+	workdirJoin := filepath.ToSlash(workdir.Join("a/file*.txt"))
+	assert.Equal(t, []string{workdirJoin}, expected)
+}
+
+func TestExpandSourceWithRawFlagFalse(t *testing.T) {
```

Comment:
```suggestion
func TestExpandSourceWithoutRawFlag(t *testing.T) {
```

Reply:
changed to `TestExpandSourceWithoutRawFlag`